### DISCUSSION TITLE
Add to apple calendar error

### DISCUSSION THREAD

**Discussion by Deep777** [add2cal/add-to-calendar-button#464] [Q&A] [answered]
Hi Guys,

Please help, I am using the add to calendar button package and for some reason getting an error adding to the apple calendar.... Google and outlook work fine however Apple is playing up.  I am using React, Please see the config I have setup below:

config console log
description: "www.test.com/testing"
endTime: "13:00"
iCalFileName: "visit"
name: "mr bloggs"
options: ["Apple"] (1)
startDate: "2021-08-26"
startTime: "08:00"
timeZone: "Europe/London"

On safari i get the attached screenshot error.

<img width="954" alt="Screenshot 2023-09-13 at 17 37 16" src="https://github.com/add2cal/add-to-calendar-button/assets/20840865/27b43a94-4cfb-4fbe-af7d-d5961aa10c64">

**Comment by jekuer** [comment]
Can you provide us with some example on codepen or Stackblitz? Like the one here: https://stackblitz.com/edit/add-to-calendar-button-react-typescript?file=App.tsx

This helps to reproduce and debug .

**Comment by Deep777** [comment]
Hi jekur 

Here is a link of the way i have implemented this calendar, outlook and google work fine, however if you add Caltypes.Apple on line 21 in AppleCalendarExample component the error appears.  

https://stackblitz.com/edit/react-starter-typescript-9ayha9?file=App.tsx

Appreciate your help on this

**Reply by jekuer** [reply]
Thanks!
Confirmed.

**Reply by jekuer** [reply]
-> #466

**Comment by jekuer** [comment, marked as answer]
@Deep777 , 
Should be fixed with v2.4.2

**Reply by Deep777** [reply]
Thank you @jekuer for your quick turn around on this issue.

The issue has been partly fixed, in that I am able to save to the Apple calendar now however an error occurs on the screen as shown in the screenshot.

You can also test using, https://stackblitz.com/edit/react-starter-typescript-9ayha9?file=App.tsx

Appreciate your help on this.

![Screenshot 2023-09-18 at 12 54 01](https://github.com/add2cal/add-to-calendar-button/assets/20840865/22bc4045-a772-4e61-acfa-a488e86f60c0)

**Reply by jekuer** [reply]
Damn. Missed one tiny detail. Will fix soon.

**Reply by jekuer** [reply]
@Deep777 , I hope the latest fix solves it!

**Reply by Deep777** [reply]
Yes working now, Thanks again!!! :o)
